packages/notebook/src/widget.ts

```diff
@@ -1621,6 +1621,13 @@ export class Notebook extends StaticNotebook {
     return this._selectionChanged;
   }
 
+  get cellsPasted(): ISignal<
+    this,
+    { action: 'copy' | 'cut' | 'paste' | null; count: number }
```

Comment:
Should this be an interface with documentaiton for each of the members? It is not immediately obvious what either the `action` nor `count` represents. Should `action` be called `lastInteraction` or `previousInteraction` or `previousAction` instead?